spring-boot/src/main/java/com/people/peoplemanager/repository/PersonRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.people.peoplemanager.repository;
+
+import com.people.peoplemanager.model.Person;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface PersonRepository extends JpaRepository<Person, UUID> {
```

Comment:
Pra ficar consistente com o Controller
```suggestion
public interface PeopleRepository extends JpaRepository<Person, UUID> {
```